src/migrations/cluster-store/3.6.0-beta.1.ts

```diff
@@ -48,13 +43,8 @@ export default migration({
             printLog(`migrating ${cluster.preferences.icon} for ${cluster.preferences.clusterName}`)
             const iconPath = cluster.preferences.icon.replace("store://", "")
             const fileData = fse.readFileSync(path.join(userDataPath, iconPath));
-            const { mime = "" } = makeSynchronous(getFileType)(fileData);
-
-            if (!mime) {
-              printLog(`mime type not detected for ${cluster.preferences.clusterName}'s icon: ${iconPath}`)
-            }
 
-            cluster.preferences.icon = `data:${mime};base64, ${fileData.toString('base64')}`;
+            cluster.preferences.icon = `data:;base64,${fileData.toString('base64')}`;
```

Comment:
you removed a space which I think the cluster-store unit test expects?

Reply:
hmm, probably looking at the wrong thing, it also still expects the mimetype

Reply:
Unit test failed, heh, I think I was right the first time.

Reply:
Yes you were, I forgot to remove the space from the tests (the tests are wrong)